showCrime/showCrime/settings.py

```diff
@@ -224,3 +233,23 @@ def generate_file_handler(filename):
 SERVER_EMAIL = env('SERVER_EMAIL')
 EMAIL_CONFIG = env.email_url('EMAIL_URL')
 vars().update(EMAIL_CONFIG)
+
+###################
```

Comment:
What is the motivation for printing settings every time the application starts/restarts? This is a potential security risk, as it is far too easy to print the `SECRET_KEY`. If you're developing locally and need to see the settings, you can [open a shell and view the settings](https://stackoverflow.com/questions/6369697/how-do-i-print-out-the-contents-of-my-settings-in-a-django-shell).